docs/installation.rst

```diff
@@ -86,10 +86,7 @@ You should now be able to successfully ``import webbpsf`` in a Python session.
 
 .. Note::
 
-   **For STScI Users Only:** Users at STScI may access the required data files from the Central Storage network. Set the following environment variables in your ``bash`` shell. (You will probably want to add this to your ``.bashrc``.) ::
-
-      export WEBBPSF_PATH="/grp/jwst/ote/webbpsf-data"
-      export PYSYN_CDBS="/grp/hst/cdbs"
+   **For STScI Users Only:** Please note, we are no longer providing users at STScI a location with the required data files on the Central Storage network. Therefore, to run WebbPSF, you must download these files and tell WebbPSF where to find them using the WEBBPSF_PATH environment variable. ::
```

Comment:
minor RST syntax comment. You don't need the `::` at the end of the paragraph now, since it's no longer followed by a code block to format.